google/cloud/storage/benchmarks/storage_throughput_benchmark.cc

```diff
@@ -176,45 +177,42 @@ int main(int argc, char* argv[]) {
 
 namespace {
 char const* ToString(OpType type) {
-  static char const* kOpTypeNames[] = {"READ", "WRITE", "CREATE", "DELETE",
-                                       "LAST"};
+  static char const* kOpTypeNames[] = {"READ", "WRITE", "CREATE", "LAST"};
```

Comment:
Should anyone be calling these with `XXX_LAST`?

Reply:
No. Removed the `_LAST` enum values and used `switch` to be able to detect any missing cases.